cypress/fixtures/list_of_articles_with_published.json

```diff
@@ -0,0 +1,38 @@
+    "image": "https://image.freepik.com/free-photo/sailing-boats-yachts-pier-stockholm-front-city-center_72229-307.jpg",
+    "written_by": "George",
+    "created_at": "2019-05-24T09:56:01.168Z",
+    "published": false,
+    "country": "Sweden",
+    "city": "Stockholm" 
+  },
+  {
+    "id": 2,
+    "title": "A Day in Paris",
+    "ingress": "How to spend 24 hours in Paris",
+    "body": "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?",
+    "image": "https://image.freepik.com/free-photo/paris-city-view-from-high_1160-763.jpg",
+    "written_by": "George",
+    "created_at": "2019-05-24T09:56:01.168Z",
+    "published": false,
+    "country": "France",
+    "city": "Paris" 
+  },
+  {
+    "id": 3,
+    "title": "A Day in Rome",
+    "ingress": "How to spend 24 hours in Rome",
+    "body": "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?",
+    "written_by": "George",
+    "created_at": "2019-05-24T09:56:01.168Z",
+    "image": "https://image.freepik.com/free-photo/roman-coliseum-seen-from-afar_1219-15.jpg",
+    "published": true,
+    "country": "Italy",
+    "city": "Rome" 
+  }
```

Comment:
Shouldn't we try to incorporate this mock-file in the list_of_articles fixture... So we don't have duplicates?

Reply:
I've tried that, the test doesnt pass, I have no idea why it doesnt work